Kconfig.zephyr

```
@@ -27,6 +27,9 @@ osource "$(KCONFIG_BINARY_DIR)/Kconfig.soc.defconfig"
 osource "soc/$(ARCH)/*/Kconfig.defconfig"
 # This loads the toolchain defconfigs
 osource "$(TOOLCHAIN_KCONFIG_DIR)/Kconfig.defconfig"
+#
+# This loads the testsuite defconfig
+osource "subsys/testsuite/Kconfig.defconfig"
```

no need for `osource` here, as this file is expected to always be available or something is wrong.

```suggestion
source "subsys/testsuite/Kconfig.defconfig"
```